recipe/meta.yaml

```diff
@@ -1,60 +1,57 @@
 {% set name = "pydantic" %}
-{% set version = "2.5.3" %}
-{% set repo_url = "https://github.com/pydantic/pydantic" %}
-{% set docs_url = "https://docs.pydantic.dev/latest/" %}
+{% set version = "2.8.2" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: b3ef57c62535b0941697cce638c08900d87fcb67e29cfa99e8a68f747f393f7a
-  patches:
-    - 0001-do-not-depend-on-hatch-fancy-pypi-readme.patch
```

Comment:
The file `0001-do-not-depend-on-hatch-fancy-pypi-readme.patch` is already redundant and must be removed.

Reply:
Thanks! Removed!